packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomGroupHeaders.Example.tsx

```diff
@@ -56,10 +56,15 @@ export class DetailsListCustomGroupHeadersExample extends React.Component {
     );
   }
 
-  private _onClick(props: IGroupDividerProps): () => void {
+  private _onToggleSelectGroup(props: IGroupDividerProps): () => void {
     return () => {
       props.onToggleSelectGroup!(props.group!);
     };
   }
 
+  private _onToggleCollapse(props: IGroupDividerProps): () => void {
+    return () => {
+      props!.onToggleCollapse!(props!.group!);
```

Comment:
`!` on `props!` and `props!.group!` shouldn't be needed since TypeScript will catch undefined values being passed in